doc/version_history.rst

```diff
@@ -6,6 +6,20 @@
 Version History
 ###############
 
+v0.31.0
+-------
+
+* `CommandTelemetryClient`: inherit from `lsst.ts.tcpip.Client`, which requires ts_tcpip 1.0.
```

Comment:
inherit --> inherits

Reply:
I prefer the active voice for release notes.